R/database.R

```diff
@@ -1665,17 +1665,17 @@ get_lymphgen = function(these_samples_metadata,
       pivot_wider(names_from="Feature",values_from="present")
     feat_st2 = mutate(feat_st2_genes,Class="ST2")
 
-    all_n1 = separate(lg_tidy,col="N1.Features",into=c(paste0("Feature_MCD_",seq(1:15))),sep=",") %>%
+    all_n1 = suppressWarnings(separate(lg_tidy,col="N1.Features",into=c(paste0("Feature_MCD_",seq(1:15))),sep=",")) %>%
       pivot_longer(starts_with("Feature_"),values_to = "MCD") %>% dplyr::filter(!is.na(MCD)) %>% pull(MCD) %>% unique()
     all_n1_genes = str_remove(all_n1,"_.*") %>% unique()
```

Comment:
Hard to review this with all the whitespace changes. Can you tell me what line numbers actually changed ?